raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -47,7 +48,7 @@ contract NettingChannelContract {
         (success, balance) = data.deposit(amount);
 
         if (success == true) {
-            ChannelNewBalance(data.token, msg.sender, balance, data.opened);
+            ChannelNewBalance(data.token, msg.sender, balance, block.number);
```

Comment:
we shouldn't need the block.number here, https://github.com/raiden-network/raiden/issues/450